src/tokens.js

```diff
@@ -59,6 +60,52 @@ export function unsetTokens() {
   unsetUser();
 }
 
+/**
+ * Client-side check:
+ * Determine whether the access token is present and unexpired
+ * @returns {Boolean}
+ */
+export function isAccessTokenLocallyValid() {
+  return isJwtLocallyValid(store.tokens.accessToken);
+}
+
+/**
+ * Client-side check:
+ * Determine whether the refresh token is present and unexpired
+ * @returns {Boolean}
+ */
+export function isRefreshTokenLocallyValid() {
+  return isJwtLocallyValid(store.tokens.refreshToken);
+}
+
+/**
+ * Determine whether a user is logged in by checking their
+ * JWT access token and, if invalid, refreshing it and checking
+ * again.
+ * @returns {Boolean}
+ */
+export async function isLoggedIn() {
```

Comment:
Hmm, my instinct is that `isFoo` is a read operation without side effects that returns a plain Boolean. I'd be surprised to call `isLoggedIn` and find that it makes a network request and returns a Promise.

But I haven't thought of a better name that keeps the interface simple... `checkIfLoggedIn` is the best I've got so far.

Reply:
`getUserStatus` could be another idea, could return e.g. `{ isLoggedIn: true, user: { ... } }`.

Reply:
I agree with the sentiment. It started as a boolean and then evolved into a promise, so the name should probably change. I'll put something up shortly.

Reply:
Ended up going with `getSession`, which can evolve to give lots of information about the session (duration, original location, number of times refreshed, factor(s) used for login, etc.). For now, usage looks like:

```
const { isLoggedIn } = await Userfront.getSession();
```

@moroshko I published this as `@userfront/core@0.4.3-beta.0` pending your feedback.